models/project/board.go

```diff
@@ -219,6 +219,19 @@ func GetBoard(ctx context.Context, boardID int64) (*Board, error) {
 	return board, nil
 }
 
+// GetBoard fetches the current default board of a project
+func GetDefaultBoard(ctx context.Context, projectID int64) (*Board, error) {
```

Comment:
This is duplicated with `func (p *Project) getDefaultBoard(ctx context.Context) (*Board, error)`, maybe we should merge them as one.